website/content/resources-hub/_index.md

```diff
@@ -57,12 +51,10 @@ This comprehensive session covers Red Hat's Secure Software Development Lifecycl
 
 *Features Ralph's hands-on demonstration showing how conforma works within Konflux in practice.*
 
-### 4. Achieving Simplicity and Security with Konflux
-**Format:** TechGenie Presentation  
+## Achieving Simplicity and Security with Konflux
+**Speaker:** Dheeraj Singh Jodha  
+**Format:** Virtual Presentation  
+**Event:** TechGenie  
 **Link:** [Watch on YouTube](https://www.youtube.com/watch?v=sR8U5-UPGQs)
```

Comment:
Perhaps an embedded video player would look nice as well as (or instead of) the plain link.

Reply:
I think so too. It would make users more likely to click play, and would also make the page look full-er.